tox.ini

```diff
@@ -10,7 +10,7 @@ ignore = P102,B311,W503,E226,S311, E221
          D301,D302,
          # Docstring Content
          D401,D402,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D416,D417
-exclude = __pycache__, venv, .venv, .git, setup.py, __init__.py
+exclude = __pycache__, venv, .venv, .git, setup.py, __init__.py, act
```

Comment:
what's `act` ?

Reply:
Act is that offline CI testing tool you advised me to use. It makes a lot of badly linted files.